### PR TITLE
 Itunes Fix- goto not available

### PR DESCRIPTION
**Pull request by Darshan-Chauhan**
Hi Team,

This PR contains fix for iTunes: GoTo not available.
I've just commented the code in case we want to enable this feature in future.

Please review.
Thanks

### CODE REVIEW

File: app/templates/sticky_footer.hbs [hunk 1]
```diff
@@ -22,7 +22,7 @@
             <i>{{fa-icon 'cube'}}</i>{{t "index.items"}}
           {{/link-to}}
 
-          <li><i>{{fa-icon 'qrcode'}}</i>{{t "index.goto"}}</li>
+          {{!-- <li><i>{{fa-icon 'qrcode'}}</i>{{t "index.goto"}}</li> --}}
```
Comment (abhinav098) [inline app/templates/sticky_footer.hbs]: Why have we commented out this line ?

Comment (Darshan-Chauhan) [reply app/templates/sticky_footer.hbs]: Because it doesn't redirect anywhere

Comment (abhinav098) [reply app/templates/sticky_footer.hbs]: oh, okkay. So Is that's why the ios-build was failing ?

Comment (Darshan-Chauhan) [reply app/templates/sticky_footer.hbs]: No, that's why they rejected our app on iTunes

Comment (abhinav098) [reply app/templates/sticky_footer.hbs]: Okay great.. excellent work 👍

**Comment by namrataukirde** [comment]
👍

**Pull request merged**